sources/packages/backend/workflow/BPMN/camunda-8/parttime-configure-disbursement.bpmn

```diff
@@ -52,25 +52,16 @@
+          <zeebe:output source="=[&#10;  {&#10;    disbursementDate: disbursmenentScheduleDate1,&#10;    negotiatedExpiryDate: disbursmenentScheduleDate1,&#10;    disbursements: [&#10;      {&#10;        valueCode: &#34;CSLP&#34;,&#10;        valueType: &#34;Canada Loan&#34;,&#10;        valueAmount: ceiling(finalFederalAwardNetCSLPAmount,0),&#10;        awardEligibility: true&#10;      },&#10;      {&#10;        valueCode: &#34;CSGP&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: ceiling(finalFederalAwardNetCSGPAmount, 0),&#10;        awardEligibility: awardEligibilityCSGP&#10;      },&#10;      {&#10;        valueCode: &#34;CSGD&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: ceiling(finalFederalAwardNetCSGDAmount, 0),&#10;        awardEligibility: awardEligibilityCSGD&#10;      },&#10;      {&#10;        valueCode: &#34;CSPT&#34;,&#10;        valueType: &#34;Canada Grant&#34;,&#10;        valueAmount: ceiling(finalFederalAwardNetCSPTAmount, 0),&#10;        awardEligibility: awardEligibilityCSPT&#10;      },&#10;      {&#10;        valueCode: &#34;BCAG&#34;,&#10;        valueType: &#34;BC Grant&#34;,&#10;        valueAmount: ceiling(finalProvincialAwardNetBCAGAmount, 0),&#10;        awardEligibility: awardEligibilityBCAG&#10;      },&#10;      {&#10;        valueCode: &#34;SBSD&#34;,&#10;        valueType: &#34;BC Grant&#34;,&#10;        valueAmount: ceiling(finalProvincialAwardNetSBSDAmount, 0),&#10;        awardEligibility: awardEligibilitySBSD&#10;      }&#10;    ]&#10;    [item.awardEligibility != false]&#10;  }&#10;]" target="disbursementSchedules" />
```

Comment:
Why are we explicitly setting the ceiling scale to 0 ?

Reply:
sorry, forgot to remove it. Done. Thanks!